pom.xml

```diff
@@ -77,7 +77,7 @@
         <dependency>
             <groupId>jline</groupId>
             <artifactId>jline</artifactId>
-            <version>2.11</version>
+            <version>2.12.1</version>
```

Comment:
I should mention that 2.12 had a bug with console input, might require a rewrite of the console :)